rust_snuba/src/strategies/clickhouse.rs

```diff
@@ -166,15 +163,14 @@ impl ClickhouseClient {
         }
     }
 
-    fn chunk(&self, rows: Vec<u8>) -> Vec<Vec<u8>> {
+    fn chunked_body(rows: &[u8]) -> Vec<Result<Box<[u8]>, std::io::Error>> {
```

Comment:
Is this a `Vec<Result<…>>` so it lines up with `wrap_stream` below?

Reply:
Yes exactly